apps/nextra/pages/en/build/create-aptos-dapp.mdx

```diff
@@ -2,89 +2,63 @@
 
-## **What is create-aptos-dapp?**
+# create-aptos-dapp
 
-`create-aptos-dapp` simplifies the initial setup and configuration process, provides a modern development workflow, gives pre-made e2e dapp templates, and offers a range of benefits that save time and effort. Using `create-aptos-dapp` enables developers to build dapps quicker on Aptos.
+`create-aptos-dapp` builds a template project for dapp developers to easily create a front-end and a smart contract on the Aptos network.
 
-## **Why use create-aptos-dapp?**
+## Why use create-aptos-dapp?
 
-- **Template Setup**: `create-aptos-dapp` generates predefined end-to-end dapp templates and configuration files for you. It saves manual setup of the project structure, which can be time-consuming and error-prone.
-- **Dependencies Management**: `create-aptos-dapp` manages project dependencies for you. It generates a npm (or pnpm, etc.) package with the required packages. This ensures that the libraries used by your project are compatible.
-- **Move Directory:** `create-aptos-dapp` generates a `move` directory that includes the basic structure for Move modules (smart contracts). Additionally, it adds a basic Move module and associated files.
-- **Best Practices**: `create-aptos-dapp` incorporates best practices and structure recommendations to develop for the Aptos network. This ensures that your project starts with a solid foundation.
-- **Built-in Move Commands**: `create-aptos-dapp` includes built-in commands for common tasks, such as initializing the Move compiler, compiling, and publishing smart contracts on-chain. This abstracts Move development workflows for the average dapp developer.
+- **Templated Setup**: `create-aptos-dapp` generates predefined end-to-end dapp templates and configuration files for you. It saves manual setup of the project structure, which can be time-consuming and error-prone.
+- **Move Directory:** `create-aptos-dapp` generates a `move` directory that includes the basic structure for Move modules (smart contracts).
+- **Best Practices**: `create-aptos-dapp` incorporates best practices and structure recommendations to develop for the Aptos network.
+- **Built-in Move Commands**: `create-aptos-dapp` includes built-in commands for common tasks, such as initializing the Move compiler, compiling, and publishing smart contracts on-chain.
 
-## What tools create-aptos-dapp utilizes?
+## Using `create-aptos-dapp`
 
-- React framework
-- Vite development tool
-- shadcn/ui + tailwind for styling
-- Aptos TS SDK
-- Aptos Wallet Adapter
-- Node based Move commands
+<Steps>
+### Navigate to the directory you want to work in. 
 
-## How to use create-aptos-dapp?
+```bash filename="Terminal"
+cd your/workspace
+```
 
-To create an Aptos dapp, open your terminal, `cd` into the directory you’d like to create the dapp in, and run the following command:
+### Install `create-aptos-dapp`.
 
-<Tabs items={['npm', 'pnpm', 'yarn']}>
+<Tabs items={['npx', 'pnpx', 'yarn']}>
```

Comment:
```suggestion
<Tabs items={['npm', 'pnpm', 'yarn']}>
```

Reply:
The command is `npx` and `pnpx`, so I think the tab titles should match. 

When I was going through this tutorial myself, I found the mismatch confusing because while I did have `pnpm` installed, I did not have `pnpx` installed yet.